### PR TITLE
proof read for MEASUREMENT_INCORPORATION

### PR DESCRIPTION
**Pull request by donquxiote**
PR for #227 , covers pages 1252 to 1261

Most of the changes were around whitespace in comment blocks. Biggest change to the code was many instances of `ZI` which (as far as I could tell) should be `Z1`

### CODE REVIEW

**Comment by wopian** [comment]
From the scans, only the function name (`Z123`) and the comment referencing it are printed with a `1` (one), while it is `ZI` (letter i) everywhere else - the only difference between `1` and `I` in the scans is the top crossbar not continuing all the way across. E.g `IX1`:

image

**Comment by donquxiote** [comment]
fixed the `Z1` back to `zi` with the letter

File: Comanche055/MEASUREMENT_INCORPORATION.agc [hunk 1]
```diff
@@ -87,29 +86,29 @@ Z123		VLOAD	MXV*
...
-			ZI 	+18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108) FIRST PASS
-		STORE	ZI 	+18D,2	# ZI THEN Z2 THEN Z3
+			W +162D,1
+			ZI +18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108)FIRST PASS
+		STORE	ZI +18D,2	# ZI THEN Z2 THEN Z3
 		TIX,1
 			INCOR1
 INCOR1		TIX,2	BON
-			Z123		# LOOP FOR Z1,Z2,Z3
+			Z123		# LOOP FOR ZI,Z2,Z3
```
Comment (chrislgarry) [inline Comanche055/MEASUREMENT_INCORPORATION.agc]: screen shot 2018-11-12 at 17 58 06

It looks like it should be Z1 here.

Comment (chrislgarry) [reply Comanche055/MEASUREMENT_INCORPORATION.agc]: @wopian looks like a 1 to me

Comment (wopian) [reply Comanche055/MEASUREMENT_INCORPORATION.agc]: Yeah, I forgot about that one. 😅

File: Comanche055/MEASUREMENT_INCORPORATION.agc [hunk 2]
```diff
@@ -87,29 +86,29 @@ Z123		VLOAD	MXV*
 		VLOAD
 			BVECTOR +12D	# BVECTOR (2)
 		MXV*	VAD*
-			W 	+162D,1
-			ZI 	+18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108) FIRST PASS
-		STORE	ZI 	+18D,2	# ZI THEN Z2 THEN Z3
+			W +162D,1
+			ZI +18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108)FIRST PASS
```
Comment (wopian) [inline Comanche055/MEASUREMENT_INCORPORATION.agc]:
```diff
- PASS
+ PAS
```

http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1253.jpg

**Comment by wopian** [comment]
Thank you for helping proof read the scans 🎉

**Pull request merged**